EasyPost/Exception.cs

```diff
@@ -4,11 +4,13 @@ namespace EasyPost {
     public class HttpException : Exception {
         public int StatusCode;
         public string Code;
+        public Error Error;
 
         public HttpException(int statusCode, string code, string message)
             : base(message) {
             StatusCode = statusCode;
             Code = code;
+            Error = Error.Load<Error>(message);
```

Comment:
The signature of this function should change to `public HttpException(int statusCode, string code, string message, List<Error> errors)`. In `EasyPost/Client.cs` the you should call `Error.Load<Error>` on the errors map of the `errors` key. All error objects from EasyPost will be of the following format:

```js
{
  "error": {
    "code": "FOO.BAR",
    "message": "foobar",
    "errors": [
      {
        field: "foo",
        message: "invalid"
      },
      {
        field: "bar",
        message: "invalid"
      }
    ]
  }
}
```

https://www.easypost.com/docs/api#errors
https://www.easypost.com/errors-guide.html

Reply:
The error class should include 'code' & 'suggestion'. 

The following is a sample from api#errors doc. 

` "verifications": {
    "delivery": {
      "success": false,
      "errors": [
        {
          "suggestion": null,
          "code": "E.ADDRESS.NOT_FOUND",
          "field": "address",
          "message": "Address not found"
        },
        {
          "suggestion": null,
          "code": "E.HOUSE_NUMBER.MISSING",
          "field": "street1",
          "message": "House number is missing"
        }
      ],
      "details": {}
    }
  }`